docs/conversion.md

```diff
@@ -35,7 +35,9 @@ __Glossary:__
 | deploy: update_config  | -  | -  | n  |                                                             |                                                                                                                |
 | deploy: resources      | -  | -  | ✓  | Containers.Resources.Limits.Memory                          | Support for memory but not CPU                                                                                 |
 | deploy: restart_policy | -  | -  | ✓  | Pod generation                                              | This generated a Pod, see the [user guide on restart](http://kompose.io/user-guide/#restart)                   |
-| deploy: labels         | -  | -  | n  |                                                             |                                                                                                                |
+| deploy: labels         | -  | -  | ✓  |                                                             |                                                                                                                |
+| deploy: cpus           | -  | -  | ✓  |                                                             |                                                                                                                |
+| deploy: memory         | -  | -  | ✓  |                                                             |                                                                                                                |
```

Comment:
remove "deploy: memory". this is covered under "deploy: resource" which is already checkmarked.